.eslintrc.js

```diff
@@ -7,7 +7,12 @@ module.exports = {
     'not-an-aardvark/node',
     'plugin:node/recommended',
     'plugin:self/all',
+    'plugin:unicorn/recommended',
   ],
+  parserOptions: {
+    ecmaVersion: 2021,
+    sourceType: 'script',
+  },
```

Comment:
this should have been set in `pugin:node/recommended`?

Reply:
eslint-plugin-unicorn sets the `sourceType to `module` so we need to change it back to `script` here.

Reply:
well, we can remove it when rewrite it in esm. :)